### PR TITLE
added task and updated readme

### PR DESCRIPTION
**Pull request by gestap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,26 @@
-# Base template for Gulp
+# Sibling tree
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Create a tree that clicks on a headline hides-shows children:
```
Comment (roman-mate) [inline README.md]:
```suggestion
Create an expandable tree.
```

File: README.md [hunk 2]
```diff
@@ -1,79 +1,26 @@
...
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
+Requirements: 
 
-All packages will be installed after executing the script above.
+- Use delegation.
```
Comment (roman-mate) [inline README.md]: Add requirements that clicking on an item should expand it's content.

File: README.md [hunk 3]
```diff
@@ -1,79 +1,26 @@
...
+- Clicking outside the title text (in an empty place) should not do anything.
```
Comment (roman-mate) [inline README.md]: double negation in english is not allowed. 

"should not do anything" -> "should do nothing".

File: README.md (hunk 1 above)
Comment (gestap) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (gestap) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (gestap) [reply README.md]: done

File: src/index.html [hunk 4]
```diff
@@ -6,8 +6,41 @@
...
+    <ul class="tree">
+      <li>Fruit</li>
+      <ul>
+        <li>Red</li>
+        <ul>
+          <li>Cherry</li>
+          <li>Strawberry</li>
+          <li>Watermelon</li>
+          <li>Pomegranate</li>
+        </ul>
```
Comment (mgrinko) [inline src/index.html]: why didn't you wrap `ul` with `li`???
It is not valid HTML

File: README.md [hunk 5]
```diff
@@ -1,79 +1,27 @@
-# Base template for Gulp
+# Sibling tree
```
Comment (mgrinko) [inline README.md]: why `Sibling`?
Originally it was `Sliding`

File: README.md [hunk 6]
```diff
@@ -1,79 +1,27 @@
...
-```bash
-$ npm install
-```
+Requirements: 
 
-All packages will be installed after executing the script above.
+- Use delegation.
+- Clicking on the headline should hide or show elements (as in the screenshots).
+- Clicking outside the title text (in an empty place) should do nothing.
+- When you hover over the title - it becomes bold, implemented through CSS.
```
Comment (mgrinko) [inline README.md]: hover over sounds like a duplication

File: src/index.html [hunk 7]
```diff
@@ -6,8 +6,41 @@
...
+        <li>Freshwater fish</li>
+        <ul>
+          <li>Oscar</li>
+          <li>Mollies</li>
+          <li>Goldfish</li>
+        </ul>
+      </ul>
+    </ul>
+
     <script type="text/javascript" src="scripts/main.js"></script>
```
Comment (mgrinko) [inline src/index.html]: why do we put the `script` at the end of the `body` instead of `head`???

File: README.md (hunk 5 above)
Comment (gestap) [reply README.md]: Знімок екрана  о 11 21 24 дп
I copied the task name from the spreadsheet

Comment (roman-mate) [reply README.md]: lol, let's rename :)

Comment (gestap) [reply README.md]: @roman-mate what do you think about it?

File: src/index.html (hunk 4 above)
Comment (gestap) [reply src/index.html]: fixed

File: README.md (hunk 5 above)
Comment (gestap) [reply README.md]: changed project name

File: README.md (hunk 6 above)
Comment (gestap) [reply README.md]: changed

File: src/index.html (hunk 7 above)
Comment (gestap) [reply src/index.html]: moved it to the `head`

**Comment by gestap** [comment]
@mgrinko, I need your approve

File: README.md (hunk 5 above)
Comment (mgrinko) [inline README.md]: PLease rename also here

**Pull request merged**